src/platform/site-wide/user-nav/sass/user-nav.scss

```diff
@@ -269,4 +269,11 @@ span.sidelines {
 
 #help-menu {
   width: 21rem;
+}
+
+#address-validation-warning {
```

Comment:
I'm thinking that we should make this change to the AlertBox component itself unless we are sure that this is the only instance of an AlertBox in a modal that has this issue.

Reply:
@erikphansen do we want to make this change for all alert boxes? Or do we want this isolated to the only alert box that (we know of) thats experiencing this issue?

Reply:
Yeah the same thing happens in the Direct Deposit payment info modal

<img width="446" alt="Screen Shot 2020-02-14 at 9 47 26 AM" src="https://user-images.githubusercontent.com/20728956/74554755-1abe3c00-4f0f-11ea-8dc0-50de8da9b8af.png">

Reply:
@erikphansen Righto, Ill move this over to formation